### DISCUSSION TITLE
SD Maid pro question

### DISCUSSION THREAD

**Discussion by obedientgoyim** [d4rken-org/sdmaid-se#49] [General]
I bought the original SD Maid, am I not able to use this pro upgrade for SD Maid SE?

**Comment by d4rken** [comment]
It's two separate apps with their own pro upgrade. It's not transferable.

Added a small FAQ entry about this:
https://github.com/d4rken-org/sdmaid-se/wiki/FAQ#transfer-pro-status-from-sd-maid-1

**Reply by obedientgoyim** [reply]
All good thanks for the reply!
